### PR TITLE
document conditional jobs.exclude and jobs.allow_failures

### PR DESCRIPTION
**Pull request by svenfuchs**

### CODE REVIEW

File: user/conditional-builds-stages-jobs.md [hunk 1]
```diff
@@ -21,7 +21,9 @@ Build requests that do not match the condition will not generate a build, but wi
 
 ## Conditional Stages
 
-Configure Travis CI to only include stages when certain conditions are met, such as only including the stage on the master branch. Stages that do not match the given condition are silently skipped.
+Configure Travis CI to only include stages when certain conditions are met. Stages that do not match the given condition are silently skipped.
+
+For example, this allows the deploy stage to run only on the master branch:
```
Comment (BanzaiMan) [inline user/conditional-builds-stages-jobs.md]: I would want `master` be either stylized or quoted. (I prefer the former.)

File: user/conditional-builds-stages-jobs.md [hunk 2]
```diff
@@ -35,7 +37,9 @@ Stages that do not match the condition will be skipped silently.
 
 ## Conditional Jobs
 
-Configure Travis CI to only include jobs when certain conditions are met, such as only including a job on the master branch. Jobs that do not match the given condition are silently skipped.
+Configure Travis CI to only include jobs when certain conditions are met.
+
+For example, this includes the listed job only to builds on the master branch:
```
Comment (BanzaiMan) [inline user/conditional-builds-stages-jobs.md]: ditto

File: user/conditional-builds-stages-jobs.md [hunk 3]
```diff
@@ -46,10 +50,44 @@ jobs:
 ```
 {: data-file=".travis.yml"}
 
-Jobs need to be listed explicitly, i.e., using `jobs.include` (or its alias `matrix.include`), in order to specify conditions for them. Jobs created via [matrix expansion](/user/customizing-the-build/#build-matrix) currently cannot have conditions.
+Jobs need to be listed explicitly, i.e., using `jobs.include` (or its alias `matrix.include`), in order to specify conditions for them. Jobs created via [matrix expansion](/user/customizing-the-build/#build-matrix) currently cannot have conditions, but they can be conditionally excluded (see the next section).
```
Comment (BanzaiMan) [inline user/conditional-builds-stages-jobs.md]: "See below", or an anchor is preferable.

File: user/conditional-builds-stages-jobs.md [hunk 4]
```diff
@@ -46,10 +50,44 @@ jobs:
...
 
 Jobs that do not match the condition will be skipped silently.
 
+## Conditionally Excluding Jobs
+
+Configure Travis CI to exclude jobs only when certain conditions are met.
```
Comment (BanzaiMan) [inline user/conditional-builds-stages-jobs.md]: The subject is missing. Perhaps "You can configure …". Also, "only" feels odd to me; it sounds better without it.

File: user/conditional-builds-stages-jobs.md [hunk 5]
```diff
@@ -46,10 +50,44 @@ jobs:
...
+
```
Comment (BanzaiMan) [inline user/conditional-builds-stages-jobs.md]: There is no need to start a new paragraph.

File: user/conditional-builds-stages-jobs.md [hunk 6]
```diff
@@ -46,10 +50,44 @@ jobs:
...
+jobs:
+  exclude:
+    - if: branch = master
+      env: TWO=two
+```
+{: data-file=".travis.yml"}
+
+## Conditionally Allowing Jobs To Fail
+
+Configure Travis CI to allow jobs to fail only when certain conditions are met.
```
Comment (BanzaiMan) [inline user/conditional-builds-stages-jobs.md]: The same comments hold here.

File: user/conditional-builds-stages-jobs.md [hunk 7]
```diff
@@ -21,7 +21,9 @@ Build requests that do not match the condition will not generate a build, but wi
 
 ## Conditional Stages
 
-Configure Travis CI to only include stages when certain conditions are met, such as only including the stage on the master branch. Stages that do not match the given condition are silently skipped.
+Configure Travis CI to only include stages when certain conditions are met. Stages that do not match the given condition are silently skipped.
```
Comment (BanzaiMan) [inline user/conditional-builds-stages-jobs.md]: The subject is missing.

File: user/conditional-builds-stages-jobs.md [hunk 8]
```diff
@@ -3,13 +3,13 @@ title: Conditional Builds, Stages and Jobs
...
 ---
 
-
-
 You can filter out and reject builds, stages and jobs by specifying conditions in your build configuration (your `.travis.yml` file).
 
 ## Conditional Builds
 
-Configure Travis CI to only trigger builds when certain conditions are met, such as only building the master branch. Any potential builds that do not meet these conditions are listed in the *Requests* tab of your repository, even though the actual build is not generated.
+Configure Travis CI to only run builds when certain conditions are met. Any builds that do not meet these conditions are listed in the *Requests* tab of your repository, even though the actual build is not generated.
```
Comment (BanzaiMan) [inline user/conditional-builds-stages-jobs.md]: The subject is missing.

File: user/conditional-builds-stages-jobs.md (hunk 1 above)
Comment (svenfuchs) [reply user/conditional-builds-stages-jobs.md]: done

File: user/conditional-builds-stages-jobs.md (hunk 2 above)
Comment (svenfuchs) [reply user/conditional-builds-stages-jobs.md]: done

File: user/conditional-builds-stages-jobs.md (hunk 3 above)
Comment (svenfuchs) [reply user/conditional-builds-stages-jobs.md]: done

File: user/conditional-builds-stages-jobs.md (hunk 4 above)
Comment (svenfuchs) [reply user/conditional-builds-stages-jobs.md]: done

File: user/conditional-builds-stages-jobs.md (hunk 5 above)
Comment (svenfuchs) [reply user/conditional-builds-stages-jobs.md]: done

File: user/conditional-builds-stages-jobs.md (hunk 6 above)
Comment (svenfuchs) [reply user/conditional-builds-stages-jobs.md]: done

File: user/conditional-builds-stages-jobs.md (hunk 7 above)
Comment (svenfuchs) [reply user/conditional-builds-stages-jobs.md]: done

File: user/conditional-builds-stages-jobs.md (hunk 8 above)
Comment (svenfuchs) [reply user/conditional-builds-stages-jobs.md]: done

**Pull request merged**